scipy/stats/tests/test_stats.py

```diff
@@ -5023,6 +5020,64 @@ def test_trim_bounds_error(self, trim):
             stats.ttest_ind([1, 2], [2, 1], trim=trim)
 
 
+class Test_ttest_CI:
```

Comment:
In retrospect, there was not really a need for this to be in a separate class. If the reviewer wants to see this elsewhere, let me know where.

Reply:
Looking at other tests, it seems like the precedent is to create a new class for testing a new argument. So, I don't think we need to change this. Maybe later we can merge all these classes to avoid create a new one everytime something new is added.